earthfile2llb/parser/EarthLexer.g4

```diff
@@ -39,7 +39,7 @@ NL: WS? COMMENT? CRLF;
 WS: [ \t] ([ \t] | LC)*;
 fragment CRLF: ('\r' | '\n' | '\r\n');
 fragment COMMENT: '#' (~[\r\n])*;
-fragment LC: '\\' [ \t]* CRLF;
+fragment LC: '\\' [ \t]* COMMENT? CRLF LC*;
```

Comment:
This actually allows 2 of the changes:
* Allows comments after a LC mark
* Allows to have several LC on sequence

Reply:
The `LC*` at the end doesn't seem right - It should already be covered by the `LC` fragment being used together with a star in `WS`.

Reply:
Yeah, it makes sense. Will try to take it out, and see how it goes